ocpp/charge_point.py

```diff
@@ -220,6 +250,7 @@ async def _handle_call(self, msg):
             # when no '_on_after' hook is installed.
             pass
 
+    @add_latency #pass argument here to add the required latency, somehow
```

Comment:
Since we depends on some inputs, I don't think that a decorator is the best approach, because is not explicit that you depend on the function to be a bounded method and that the class should contain this attributes that could colide, I think is better to create an async function and use it where we want to introduce the delay.

Reply:
why not sleep with async?